src/pages/Info.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+import { File, Info as Infoicon, Upload, BookOpen} from 'react-feather';
+
+import InfoBlock from "../components/InfoBlock";
+
+const Info = () => {
+
+  return (
+    <div className="info-page-container">
+
+      <InfoBlock
+        icon={<Infoicon className="info-page-icon"/>}
+        header={"About"}
+        infoText={"The pu-2 application is a tool developed to help in processing data prepared by the Client in the form of pu-2 forms."}
+      />
+
+      <InfoBlock
+        icon={<File className="info-page-icon"/>}
+        header={"Pu-2 form"}
+        infoText={"PU-2 form refers to personalized accounting documents and contains information about the hiring and dismissal of employees."}
+      />
+
+      <InfoBlock
```

Comment:
Let's make InfoBlock a HoC where infoText is a children component

Reply:
done